prefect_gcp/bigquery.py

```diff
@@ -52,8 +53,9 @@ async def bigquery_query(
     to_dataframe: bool = False,
     job_config: Optional[dict] = None,
     project: Optional[str] = None,
+    rows_as_dict: Optional[bool] = False,
```

Comment:
Could you add a parameter that accepts a function that can be used to transform the output (e.g. `result_transformer`)? I'd like to avoid needing to add a new parameter every time a different return type is needed.

Reply:
Done